src/Components/Web/src/Forms/InputLargeTextArea/InputLargeTextAreaInterop.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Components.Forms
+{
+    internal static class InputLargeTextAreaInterop
```

Comment:
Is there a reason not to just inline these constants in the component? They're only used from that one type, no?

Reply:
Following the existing pattern we have for `InputFile`. https://github.com/dotnet/aspnetcore/blob/main/src/Components/Web/src/Forms/InputFile/InputFileInterop.cs

Though in this case given it's just used for this component, I've inlined it 😄